src/main/webapp/js/adminSearch.es6

```diff
@@ -110,6 +110,39 @@ function adminSearchCollapseAllInstructors() {
     $('.fslink_instructor').hide();
     $('.instructorRow').attr('class', 'instructorRow');
 }
+
+function mailto(receiveremailId, subjecttype, studentName, courseName, Id, uri, status) {
```

Comment:
Perhaps rename the function to `openMailEditorWithDefaults`?

`mailto` is quite vague. It's not clear at a glance exactly what it does.